core/src/main/java/com/appintro/core/indicator/DotIndicatorController.kt

```diff
@@ -8,20 +8,19 @@ import android.widget.ImageView
 import android.widget.LinearLayout
 import androidx.core.content.ContextCompat
 import androidx.core.graphics.drawable.DrawableCompat
-import com.github.appintro.R
 
 /**
  * An [IndicatorController] that shows a list of dots and highlight the selected dot.
  * Use this when the number of page you're dealing with is not too high.
  */
 class DotIndicatorController(context: Context) : IndicatorController, LinearLayout(context) {
-    override var selectedIndicatorColor = ContextCompat.getColor(context, R.color.appintro_default_selected_color)
+    override var selectedIndicatorColor = ContextCompat.getColor(context, com.appintro.core.R.color.appintro_default_selected_color)
```

Comment:
Could you `import com.appintro.core.R as CoreR` in those cases, to improve code readibility? 

Ex. `com.appintro.core.R.color.appintro_default_selected_color`
-> `CoreR.color.appintro_default_selected_color`